lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -50,7 +50,11 @@ def container_from_native(container, default_env)
       port: container[:port],
       env: default_env.merge(env),
       memory: container[:memory],
+      memory_limit: container[:memory_limit],
+      memory_request: container[:memory_request],
```

Comment:
Maybe we should just get rid of `memory` and `cpu` altogether and put all the OR logic here in the builder?

```ruby
memory_limit: container[:memory_limit] || container[:memory]
```

Reply:
That's cleaner, I made the necessary adjustments.